interface/weno/drug-drug.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+ require_once("../globals.php");
+/*
+*   check to see if RxNorm installed
+*/ 
+	$rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = 'rxnconso';");
```

Comment:
Need to do something special here since the casing of rxnconso can break things on different OS's.
``````
$rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = '" . mitigateSqlTableUpperCase('RXNCONSO') . "'");
``````